CHANGELOG.md

```diff
@@ -34,7 +34,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Update Gradle to 7.6 ([#309](https://github.com/opensearch-project/opensearch-java/pull/309))
 - Prevent SPI calls at runtime ([#293](https://github.com/opensearch-project/opensearch-java/pull/293))
 - Add support for OpenSearch Serverless ([#339](https://github.com/opensearch-project/opensearch-java/pull/339))
-- Fix issue where completion suggestions were failing, due to being parsed as term suggestions ([#107](https://github.com/opensearch-project/opensearch-java/issues/107))
+- Fix issue where completion suggestions were failing, due to being parsed as term suggestions ([#347](https://github.com/opensearch-project/opensearch-java/pull/347))
+- Fix issue when trackTotalHits is disabled ([#372](https://github.com/opensearch-project/opensearch-java/pull/372))
```

Comment:
For next time/another PR, I think we should write a better description. For this one it would say "Fix request failing with X when trackTotalHits is disabled".

Reply:
Thank you :)